homeassistant/components/nest/__init__.py

```diff
@@ -183,19 +190,40 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
                 data=entry.data,
             )
         )
+    except ConfigurationException as err:
+        persistent_notification.async_create(
```

Comment:
We prefer to not show integration specific notifications. We can consider adding a common default notification on entry setup failure.

Reply:
OK, I will update this to just log errors for now, then have a look at what a default notification might take separately.

My real goal here is to move up the integration quality scale and meet the criteria of "Log a warning once when unavailable, log once when reconnected" from https://developers.home-assistant.io/docs/integration_quality_scale_index/#silver-%F0%9F%A5%88

What is the recommended way to handle logging once? The approach i've commonly sseen is to store global state in hass.data to track this (or via some wrapper class stored in hass.data), but wanted to ask before adding that, as it is not super satisfying.

Reply:
1. We can open an architecture issue and/or post the question on discord.
2. Yeah, store a boolean somewhere and use it to check if we've already logged.